sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPSensor.java

```diff
@@ -161,13 +172,13 @@ void scanFile(InputFile inputFile) {
       try {
         phpAnalyzer.nextFile(new PhpFileImpl(inputFile));
 
-        if (inSonarQube(context)) {
+        if (!inSonarLint(context)) {
           phpAnalyzer.getSyntaxHighlighting(context, inputFile).save();
           phpAnalyzer.getSymbolHighlighting(context, inputFile).save();
           saveNewFileMeasures(context,
             phpAnalyzer.computeMeasures(fileLinesContextFactory.createFor(inputFile)),
             inputFile);
-          if (inSonarQube(context)) {
+          if (!inSonarLint(context)) {
```

Comment:
This condition is very weird here, it's already checked a few lines above. Am I missing something?